common/accuracy/server.yml

```diff
@@ -1,5 +1,3 @@
-# server configs for https://huggingface.co/microsoft/phi-4
-model: "microsoft/phi-4"
 trust-remote-code: true
 enable-chunked-prefill: true
```

Comment:
We can remove "enable-chunked-prefill: true" as it's true by default since 0.8.4.

Also, can you please add following which is used by the PSAP team?

max-model-len: 16384

I'll file other PR(s) to remove the server.yml for models if they don't have other specific settings.